scala/scala.bzl

```diff
@@ -327,6 +327,17 @@ scala_test = rule(
   test=True,
 )
 
+def scala_version():
+  """return the scala version for use in maven coordinates"""
+  return "2.11"
+
+def scala_mvn_artifact(artifact):
```

Comment:
I'm wondering, should we really be parsing the artifact and stitching it back together? It seems like it might be more natural to just explicitly pass in the parts, but maybe I'm missing something.